specification/batch/data-plane/Microsoft.Batch/2017-09-01.6.0/BatchService.json

```diff
@@ -12584,17 +12643,70 @@
       },
       "title": "Windows operating system settings to apply to the virtual machine."
     },
+    "DataDisk": {
+      "properties": {
+        "lun": {
+          "type": "integer",
+          "format": "int32",
+          "title": "The logical unit number."
```

Comment:
Any guidance on what this should be?  Link to VMSS docs or something?  Or do we assume that anyone down at this level of VM config will know what the LUN means and what values make sense?

Reply:
The VMSS docs just say: "Specifies the Logical Unit Number of the disk in each virtual machine in the scale set."

Reply:
Which seems less than helpful

Reply:
I've tried to make it a bit clearer